app/_hub/kong-inc/ai-sanitizer/overview/_index.md

```diff
@@ -0,0 +1,23 @@
+---
+nav_title: Overview
+---
+
+The AI Sanitizer plugin for {{site.base_gateway}} helps protect sensitive information in client request bodies before they reach upstream services.
+By integrating with an external PII service, the plugin ensures compliance with data privacy regulations while preserving the usability of request data.
+It supports multiple sanitization modes, including replacing sensitive information with fixed placeholders or generating synthetic replacements that retain category-specific characteristics.
+
+Additionally, it offers an optional restoration feature, allowing the original data to be reinstated in responses when needed.
+
+{:.note}
+> This plugin extends the functionality of the [AI Proxy plugin](/hub/kong-inc/ai-proxy/) or the [AI Proxy Advanced](/hub/kong-inc/ai-proxy-advanced/) plugin, and requires an AI proxy to be configured first. 
+Check out the [AI Gateway quickstart](/gateway/latest/get-started/ai-gateway/) to get an AI proxy up and running within minutes!
+
+{:.note}
+> The AI Sanitizer plugin is relying on the [AI PII Services](https://github.com/Kong/ai-pii-service) service for identifying and sanitizing sensitive information, which can be deployed as a standalone service or docker container. Check out the [AI Sanitizer plugin](/hub/kong-inc/ai-sanitizer) for more information on how to config the plugin with the AI PII Services.
```

Comment:
```suggestion
The AI Sanitizer plugin relies on the AI PII Anonymizer Service for identifying and sanitizing sensitive information, which can be deployed as a standalone service or docker container. Check out the [AI Sanitizer plugin](/hub/kong-inc/ai-sanitizer) configuration options for more information on how to configure the plugin with the AI PII Anonymizer Service.
```
The repo is private, so we can't link to it. I don't think this needs to be a note either - it's just standard info.